packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -393,6 +394,18 @@ export default class TabAdvanced extends Component {
               labelId='vmConvertToTemplateButton'
               redirectOnSuccess='/'
             />
+            <TabButton
+              btnStyle='danger'
+              disabled={vm.power_state !== 'Halted'}
+              handler={changeVirtualizationMode}
+              handlerParam={vm}
+              icon='vm-migrate'
+              labelId={
+                vm.bootPolicy === ''
+                  ? 'vmVirtualizationPVToHVM'
+                  : 'vmVirtualizationHVMtoPV'
+              }
+            />
```

Comment:
These big buttons are starting to get ugly, also this action should be pretty rare, I think it would be best to add an action button directly beside the virtualization mode field.